src/pages/products/[id].tsx

```diff
@@ -1,9 +1,12 @@
+import React from 'react'
 import Footer from '../../components/Footer'
+import Meta from '../../components/Meta'
 import Navbar from '../../components/Navbar'
 import ProductContent from '../../components/ProductContent'
 
 const ProductView = () => (
   <div className="layout-product-view">
+    <Meta title="Product View | Cobda" />
```

Comment:
Reminder, would be nice to add a `TODO` here since the `title` should be the product name. 
<img width="362" alt="Screen Shot 2564-02-10 at 13 02 55" src="https://user-images.githubusercontent.com/32289827/107471849-69dd9200-6ba0-11eb-9fd6-74d6d584a19b.png">